include/odp/api/spec/event.h

```diff
@@ -93,6 +94,19 @@ void odp_event_types_multi(const odp_event_t event[], odp_event_type_t type[],
 int odp_event_type_multi(const odp_event_t event[], int num,
 			 odp_event_type_t *type);
 
+/**
+ * Event pool
+ *
+ * Returns handle to the pool where the event was allocated from. If the
+ * underlying event type does not have API for returning pool handle
```

Comment:
"an" before "API" would sound a bit more natural.

Reply:
Changed in V2.